src/components/BirdCard.jsx

```diff
@@ -0,0 +1,30 @@
+
+export default function BirdCard ({avian, setBirdFlu, birdFlu}) {
+
+function addBird(birb){
+    setBirdFlu([...birdFlu, birb])
+}
+    let birdCard = avian.map((birb) => {
```

Comment:
When we give a singular (meaning not plural) name to a react component, it means to us and other developers that this component is in charge of returning a single of whatever we're rendering. In this case the component **BirdCard** should only return one **BirdCard** and the mapping over the birds and creating the bird cards from them should happen either in **App.js** or in another component called **BirdCards** plural.